doc/user/QUICKSTART.md

```diff
@@ -38,22 +38,7 @@ f3d --help
 man f3d # Linux only
 ```
 
-## Supported file formats
-
-F3D supports the following file formats:
-
-* Legacy VTK (`.vtk`)
-* VTK XML (`.vtp`, `.vtu`, `.vtr`, `.vti`, `.vts`, `.vtm`)
-* Polygon File Format (`.ply`)
-* OBJ (`.obj`)
-* STL (`.stl`)
-* STEP (`.step`, `.stp`)
-* IGES (`.iges`, `.igs`)
-* Filmbox (`.fbx`): animations supported
-* Autodesk 3D Studio (`.3ds`): full scene by default
-* VDB (`.vdb`): experimental as of version 2.5.0
-
-See the [supported file formats](SUPPORTED_FILE_FORMATS.md) page for a full list of compatible file types.
+> Note: See the [supported file formats](SUPPORTED_FORMATS.md) page for the full list of file types that F3D can open.
```

Comment:
im not sure about this compared to just adding a [supported] link in the sentence above